apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -39,29 +39,32 @@
 }
 
 .c-contentPage-header--hero {
-    background-image: var(--hero-img-src);
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-position-x: 5%;
+    background-color: f.$color-support-brand-02;
 
-    @include f.media('>=narrowMid') {
-        background-position: center;
+    .c-contentPage-header-description {
+        margin-top: 0;
+        margin-bottom: f.$spacing-f;
     }
 
-    @include f.media('>=#{v.$wide-screen-content-max-width}') {
-        background-size: v.$wide-screen-content-max-width;
-        background-color: f.$color-container-strong;
-    }
-}
+    @include f.media('>=mid') {
+        display: flex;
+        gap: f.$spacing-f;
 
-.c-contentPage-header--hero .c-contentPage-header-heading,
-.c-contentPage-header--hero .c-contentPage-header-description {
-    @include f.media('>=narrowMid') {
-        max-width: 60%;
```

Comment:
I'm unsure if we want to remove this/need to replace it with something else

Reply:
keeping these declarations won't align the spacing between the content and the image to the new design.
we only should have a gap of 32px and the content should stretch the whole container.

Reply:
had a chat with @jordifornellsJET and we believe we should add a max-width of 500px for the content 💯